Formula/apache-ctakes.rb

```diff
@@ -0,0 +1,85 @@
+class ApacheCtakes < Formula
+  desc "NLP system for extraction of information from EMR clinical text."
+  homepage "https://ctakes.apache.org"
+  url "https://apache.osuosl.org/ctakes/ctakes-4.0.0/apache-ctakes-4.0.0-bin.tar.gz"
+  sha256 "37ca2b8dfe06465469ed1830fbb84dfc7bcc4295e5387d66e90a76ad2a5cdeaf"
+
+  bottle :unneeded
+
+  option "with-umls", "Install UMLS Dictionary"
```

Comment:
IMHO, this is a case where homebrew can really help the user, since you have to merge the downloaded `resources` directory with the existing one.
Also a user of this dictionary (which is likely most `ctakes` users) is unlikely to want to use homebrew without this option, because:
* a reinstall of this software will wipe their carefully merged directory, and
* if they have to download the optional directory, they might as well download the main software from the same site.